src/service/resources-impl.js

```diff
@@ -481,7 +481,6 @@ export class Resources {
     } else {
       // Create and add a new resource.
       resource = new Resource((++this.resourceIdCounter_), element, this);
-      this.buildOrScheduleBuildForResource_(resource);
```

Comment:
This is specifically because `CustomElement#completeUpgrade` will now always call `Resources#upgraded`, which does this. This call does nothing if the element hasn't been upgraded (either `StubElement` or deferred upgrade) and was called by `#completeUpgrade` after upgrade completion, or it was called sync by `CustomElement#connectedCallback` (and not called by `#completeUpgrade`).

But now that `#completeUpgrade` always calls it, we can remove. `#completeUpgrade` will be called async if upgrade isn't complete (the first case above), or sync if upgrade is sync (the second case).